exchanges/deribit/deribit_wrapper.go

```diff
@@ -208,6 +207,10 @@ func (d *Deribit) Setup(exch *config.Exchange) error {
 	if err != nil {
 		return err
 	}
+
+	// setup option decimal regex at startup to make constant checks more efficient
+	optionRegex = regexp.MustCompile(optionDecimalRegex)
```

Comment:
I don't see the benefit of having this in Setup instead of just being a package var.
Especially since it causes a race on the package var.
I'll address this in #1636

Reply:
Thank you for your timely review 😄 Thank you for addressing it